### ISSUE TITLE
Does routing work on refresh?

### ISSUE THREAD

**Issue by peterson-umoke** [ComfortAjalaOluwatimilehin/Wordpress-React-Router-Theme#1]
Good day ma,

Please by any chance does the router load the corresponding component on page refresh or does it just work from the landing page only?...

**Comment by ComfortAjalaOluwatimilehin** [comment]
Yes. It should work on refresh...if the the url / path on which it was refreshed corresponds to that given to the router ( browser router). PHP handles the rendering of the index.php regardless of path, while js handles what content gets displayed in the element with ID container.
